http-clients/netty-nio-client/src/main/java/software/amazon/awssdk/http/nio/netty/ProxyConfiguration.java

```diff
@@ -82,28 +122,30 @@ public static Builder builder() {
     public interface Builder extends CopyableBuilder<Builder, ProxyConfiguration> {
 
         /**
-         * Set the proxy host.
+         * Set the hostname of the proxy.
          * @param host The proxy host.
          * @return This object for method chaining.
          */
         Builder host(String host);
 
         /**
-         * Set the proxy port.
+         * Set the port that the proxy expects connections on.
          * @param port The proxy port.
          * @return This object for method chaining.
          */
         Builder port(int port);
 
         /**
-         * Set the proxy scheme.
+         * The HTTP scheme to use for connecting to the proxy. Valid values are {@code http} and {@code https}.
```

Comment:
Further down in the code I noted that this gets defaulted to 'http'. We should document (and test) that behavior.

Reply:
Will do.